src/components/navigation/navigation.tsx

```diff
@@ -64,7 +64,7 @@ export class Navigation {
                 </a>
                 <header class="panel-header">
                     <h1>
-                        <a href="#">{this.header}</a>
+                        <a href="#"><img alt="home" src="../collection/assets/logotype.svg"/></a>
```

Comment:
Hmm do we really want to put this here? :thinking: This is the title for the documentation, e.g. "Lime Elements" etc. Would it not be better to have the logo somewhere at the bottom, e.g. "Powered by Kompendium" or something similar

Reply:
Good question. We should talk about this. I think most potential users (companies) will have their own logotypes, which is the way they visually write their company name, brand name, etc... This is usually in some image format. But there will be also users which have to image to include, so a title will suffice for them. 

We need to make this options or configurable for them. Those who want to add an image/logo should just be able to do it, and those who prefer a simple text should also be able to do it. I don't know how we should solve this, you know best. Feel free to tweak the PR.

I also think we should add "Powered by Kompendium" watermark at the bottom of our navigation panel or somewhere, with our own logo.